app/src/main/java/com/jerboa/ui/components/comment/CommentOptionsDropdown.kt

```diff
@@ -160,6 +171,25 @@ fun CommentOptionsDropdown(
                     onReportClick(commentView)
                 },
             )
+
+            if (canMod) {
+                Divider()
```

Comment:
Having the moderation-related abilities in another divider.

Reply:
Yeah that's alright, if the popup becomes to big bc of the mod options then we'll probably have to divide these into sub menus.